scripts/var_props.py

```diff
@@ -688,6 +688,18 @@ def valid_value(self, test_value, prop_dict=None, error=False):
                     valid_val = tval
             except CCPPError:
                 valid_val = None # Redundant but more expressive than pass
+        elif self.ptype is float:
+            try:
+                tval = float(test_value)
+                if self._valid_value is not Note:
```

Comment:
I'm guessing this line is not being tested. Please add a test, then fix the typo.
Unless we are adding musical variable properties.

Reply:
:) fixed and added a test